pkg/kubelet/probe_test.go

```diff
@@ -404,16 +404,16 @@ func TestProbeContainer(t *testing.T) {
 		}
 		result, err := kl.probeContainer(&api.Pod{}, api.PodStatus{}, test.testContainer, dc.ID, dc.Created)
 		if test.expectError && err == nil {
-			t.Error("Expected error but did no error was returned.")
+			t.Error("Expected error but did no error was returned.", i)
 		}
 		if !test.expectError && err != nil {
-			t.Errorf("Expected error but got: %v", err)
+			t.Errorf("Expected error but got: %v", err, i)
```

Comment:
Not related to this PR, but it should be "Error not expected" (instead of "Expected error").